### ISSUE TITLE
sls deploy pg example get Error " Could not locate Gemfile"

### ISSUE THREAD

**Issue by misty44** [navarasu/serverless-ruby-examples#1]
Serverless: Load command interactiveCli
 Serverless: Load command config
 Serverless: Load command config:credentials
 Serverless: Load command config:tabcompletion
 Serverless: Load command config:tabcompletion:install
 Serverless: Load command config:tabcompletion:uninstall
 Serverless: Load command create
 Serverless: Load command install
 Serverless: Load command package
 Serverless: Load command deploy
 Serverless: Load command deploy:function
 Serverless: Load command deploy:list
 Serverless: Load command deploy:list:functions
 Serverless: Load command invoke
 Serverless: Load command invoke:local
 Serverless: Load command info
 Serverless: Load command logs
 Serverless: Load command metrics
 Serverless: Load command print
 Serverless: Load command remove
 Serverless: Load command rollback
 Serverless: Load command rollback:function
 Serverless: Load command slstats
 Serverless: Load command plugin
 Serverless: Load command plugin
 Serverless: Load command plugin:install
 Serverless: Load command plugin
 Serverless: Load command plugin:uninstall
 Serverless: Load command plugin
 Serverless: Load command plugin:list
 Serverless: Load command plugin
 Serverless: Load command plugin:search
 Serverless: Load command config
 Serverless: Load command config:credentials
 Serverless: Load command rollback
 Serverless: Load command rollback:function
 Serverless: Load command upgrade
 Serverless: Load command uninstall
 Serverless: Load command rubylayer
 Serverless: Load command login
 Serverless: Load command logout
 Serverless: Load command generate-event
 Serverless: Load command test
 Serverless: Load command dashboard
 Serverless: Load command output
 Serverless: Load command output:get
 Serverless: Load command output:list
 Serverless: Load command param
 Serverless: Load command param:get
 Serverless: Load command param:list
 Serverless: Load command studio
 Serverless: Load command dev
 Serverless: Invoke deploy
 Serverless: Invoke package
 Serverless: Invoke aws:common:validate
 Serverless: Invoke aws:common:cleanupTempDir
 Serverless: Packaging service...
 Serverless: Excluding development dependencies...
 Serverless: Clearing previous build ruby layer build
 Serverless: Building docker for bundle install
 Serverless: Packing the native libraries from the specified path
 Serverless: Installing gem using docker bundler
 Serverless: [DEPRECATED] The `--without` flag is deprecated because it relies on being remembered across bundler invocations, which bundler will no longer do in future versions. Instead please use `bundle config set without 'test development'`, and stop using this flag
 Could not locate Gemfile
  
   Error --------------------------------------------------
  
   Error: [DEPRECATED] The `--without` flag is deprecated because it relies on being remembered across bundler invocations, which bundler will no longer do in future versions. Instead please use `bundle config set without 'test development'`, and stop using this flag
   Could not locate Gemfile
   
       at runCommand (/app/node_modules/serverless-ruby-layer/lib/bundle.js:23:11)
       at docker (/app/node_modules/serverless-ruby-layer/lib/bundle.js:29:12)
       at ServerlessRubyBundler.bundleInstall (/app/node_modules/serverless-ruby-layer/lib/bundle.js:100:5)
   From previous event:
       at PluginManager.invoke (/usr/lib/node_modules/serverless/lib/classes/PluginManager.js:498:22)
       at PluginManager.spawn (/usr/lib/node_modules/serverless/lib/classes/PluginManager.js:518:17)
       at /usr/lib/node_modules/serverless/lib/plugins/deploy/deploy.js:122:50
   From previous event:
       at Object.before:deploy:deploy [as hook] (/usr/lib/node_modules/serverless/lib/plugins/deploy/deploy.js:102:22)
       at /usr/lib/node_modules/serverless/lib/classes/PluginManager.js:498:55
   From previous event:
       at PluginManager.invoke (/usr/lib/node_modules/serverless/lib/classes/PluginManager.js:498:22)
       at /usr/lib/node_modules/serverless/lib/classes/PluginManager.js:533:24
   From previous event:
       at PluginManager.run (/usr/lib/node_modules/serverless/lib/classes/PluginManager.js:533:8)
       at /usr/lib/node_modules/serverless/lib/Serverless.js:162:33
       at processImmediate (internal/timers.js:456:21)
   From previous event:
       at Serverless.run (/usr/lib/node_modules/serverless/lib/Serverless.js:149:74)
       at /usr/lib/node_modules/serverless/scripts/serverless.js:58:26

**Comment by navarasu** [comment]
The `bundle install` not able to locate the Gemfile. 
From which machine are you running?  Is it a docker inside the docker scenario? Are u running SLS deploy inside a virtual environment?

**Comment by misty44** [comment]
I running SLS deploy through gitlab ci

gitlab ci:
```
stages:
   - deploy
deploy_task:
   script:
      - sls plugin install -n serverless-ruby-layer
      - sls invoke -s test -f hello -l --config serverless_test.yml
```

serverless_test.yml
```service: task
frameworkVersion: ">=1.28.0 <2.0.0"
plugins:
  - serverless-ruby-layer
provider:
  name: aws
  runtime: ruby2.5
custom:
  rubyLayer:
    use_docker: true
    docker_yums:
      - postgresql-devel
    native_libs:
      - /usr/lib64/libpq.so.5
functions:
  hello:
    handler: handler.hello

**Comment by navarasu** [comment]
Yes, this is the docker inside the docker case. There is an open issue for this. Refer here https://github.com/navarasu/serverless-ruby-layer/issues/18.
I will fix this and I will spin up the next release.
Meanwhile, you can use `docker-dind` to make it to work in gitlab. https://docs.gitlab.com/ee/ci/docker/using_docker_build.html#use-docker-in-docker-workflow-with-docker-executor

**Comment by navarasu** [comment]
This issue will be tracked in this ticket navarasu/serverless-ruby-layer#18. So, Closing this issue

**Comment by navarasu** [comment]
@misty44 This issue is resolved in the 1.4.0 release. Now it supports docker inside docker case. Please update to the latest and verify.
```
npm i serverless-ruby-layer
```
